src/SprykerEco/Zed/AvalaraTaxProductOption/Persistence/Propel/Schema/spy_product_option.schema.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<database xmlns="spryker:schema-01" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed"
+          xsi:schemaLocation="spryker:schema-01 https://static.spryker.com/schema-01.xsd"
+          namespace="Orm\Zed\ProductOption\Persistence"
+          package="src.Orm.Zed.ProductOption.Persistence">
+
+    <table name="spy_product_option_value" idMethod="native" allowPkInsert="true">
```

Comment:
* Add the `ProductOption` module to dependencies because of dependecy on the database structure.
* Check the same for the `AvalaraTax` module it must have a dependency to a `Product` module.

Reply:
Resolved.